invenio_rdm_records/templates/semantic-ui/invenio_notifications/community-submission.submitted.jinja

```diff
@@ -0,0 +1,35 @@
+{# notifications/community-submission.submitted.jinja #}
+
+{% block ctx_setup %}
+    {% set submission_request = notification.context["request"] %}
+    {%- set dummy = ctx.update({
+        "submission_request": submission_request,
+        "community": submission_request["receiver"],
+        "creator": submission_request["created_by"],
+        "record": submission_request["topic"],
+        })
+    %}
+{% endblock ctx_setup %}
```

Comment:
minor: that's a bit unfortunate, I thought the context would be shared inside the template blocks... In that case, we have to make sure that this block runs first though (i.e. explicitly call the `ctx_setup` block).

I tested an alternative that also works:

```jinja
{% set request_title = notification.request.title %}
{% set recipient_name = recipient.profile.full_name %}

{% block subject %}
Request for {{ request_title }}
{% endblock subject %}
```

with this code:

```python
from flask import current_app

template = current_app.jinja_env.select_template(["zenodo_rdm/testing.jinja"])

data = {
    "notification": {"request": {"title": "This is the request title!"}},
    "recipient": {"id": 1234, "email": "test@mail.com", "profile": {"full_name": "Alex"}},
}
ctx = template.new_context(data)

print("before full render:")
print("".join(list(template.blocks["subject"](ctx))).strip())
# Prints: "Request for "  (i.e. `request_title` is not set)

# "Force" rendering the whole template (including global variables)
list(template.root_render_func(ctx))

print("after full render:")
print("".join(list(template.blocks["subject"](ctx))).strip())
# Prints: "Request for This is the request title!"  (i.e. now `request_title` is there)

print(ctx.vars)
# Prints:
#   {'request_title': 'This is the request title!', 'recipient_name': 'Alex'}
```

The unfortunate part is that we're rendering the whole template once, and then again each individual block... But that's not that expensive since it's just rendering :)

Reply:
Thank you! The less knowledge needed the better!
Updated it in the [notifications pr](https://github.com/inveniosoftware/invenio-notifications/pull/11/files#diff-0de3f101a171677955e25bf6a31171835a6e2d194b4740a8009ca109d9667e2dR49)

Reply:
This also allows to access dictionaries with the dot notation instead of having to use quotations marks :)